Ziffity/AddCustomer/Model/Import/CsvImport.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Ziffity\AddCustomer\Model\Import;
+
+use Magento\Framework\File\Csv;
+use Ziffity\AddCustomer\Profiles\ProfileInterface;
+
+class CsvImport implements ProfileInterface
+{
+    /**
+     * @var Csv
+     */
+    private $csvParser;
+    /**
+     * @param Csv $csvParser
+     */
+    public function __construct
+    (
+        Csv $csvParser
+    )
+    {
+        $this->csvParser = $csvParser;
+    }
+    /**
+     * @param $file
+     * @return array|void
```

Comment:
@msakila532 looks like it will return null | array, not most void. check this pls